### DISCUSSION TITLE
.git not top level dir in project

### DISCUSSION THREAD

**Discussion by papaja** [alirezanet/Husky.Net#63] [Q&A] [answered]
Hello,
My working directory is something like this:
root
```
├── root
│   ├── .git
│   ├── Resources
│   ├── src
│   │   ├── Project1
│   │   |   ├── husky stuff
│   │   |   ├── ...
│   │   ├── Project2

```

Here it the code that checks it:
https://github.com/alirezanet/Husky.Net/blob/9899da088d6e52873dc79e24c07418734e3a7966/src/Husky/Cli/InstallCommand.cs#L54-L61

What should I do to make husky work (be able to find .git)? I can't move .git inside of the project because entire solution is one repository.

**Comment by alirezanet** [comment, marked as answer]
Hi @papaja,

I recommend you to install husky from the root folder, `cd root` , `dotnet husky install`
but also you can specify a custom path if you want to put the husky configuration file somewhere else using `--dir` option. 
```
> dotnet husky install --help
Husky v0.5.4

USAGE
  husky install [options]

DESCRIPTION
  Install Husky hooks

OPTIONS
  -d|--dir          The custom directory to install Husky hooks. Default: ".husky".
  -p|--parallel     If True, husky resource creation will run once to prevent locks, when multiple instances are running Default: "True".
  --no-color        Disable color output Default: "False".
  -v|--verbose      Enable verbose output Default: "False".
  -h|--help         Shows help text.
```

**Reply by papaja** [reply]
Thanks for a reply, just for a clarification for me and other readers, should then everything, including dotnet new tool-manifest,
dotnet tool install Husky and task-runner.json also be placed/run in root folder?

**Reply by alirezanet** [reply]
If you put everything there it will be easier to configure and use, but as I said before you can choose to use the **custom directories** feature.
